### PR TITLE
create splash Screen

### PR DESCRIPTION
**Pull request by AhmedGhayada23**
create folder assets
use pubspec.yaml and activate assets
use class ImageApp
create splash screen
delete folder styles

### CODE REVIEW

File: lib/utils/routing/app_routes.dart [hunk 1]
```diff
@@ -1,3 +1,8 @@
 class AppRoutes {
   static const String home = '/';
+<<<<<<< HEAD
```
Comment (mohammedalian) [inline lib/utils/routing/app_routes.dart:3]: delete unused line

File: lib/utils/routing/app_routes.dart [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+  static const String onsignal = '/onsignal';
+  static const String splashScreen = '/splashScreen';
+=======
```
Comment (mohammedalian) [inline lib/utils/routing/app_routes.dart:6]: delete unused line

File: lib/utils/routing/app_routes.dart [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+>>>>>>> main
```
Comment (mohammedalian) [inline lib/utils/routing/app_routes.dart:7]: delete unused line

File: lib/utils/routing/pages.dart [hunk 4]
```diff
@@ -9,4 +11,16 @@ final List<GetPage> pages = <GetPage>[
     page: () => const HomePage(),
     binding: HomeBinding(),
   ),
+<<<<<<< HEAD
```
Comment (mohammedalian) [inline lib/utils/routing/pages.dart:14]: delete unused line

File: lib/utils/routing/pages.dart [hunk 5]
```diff
@@ -9,4 +11,16 @@ final List<GetPage> pages = <GetPage>[
...
+  GetPage(
```
Comment (mohammedalian) [inline lib/utils/routing/pages.dart:15]: delete onsignal page line

**Comment by AhmedGhayada23** [comment]
Resolve all conflicts

File: lib/features/home/presentation/pages/home_page.dart [hunk 6]
```diff
@@ -1,9 +1,8 @@
 import 'package:flutter/material.dart';
 import 'package:get/get.dart';
 import 'package:gtc_rider/core/localization/local_controller.dart';
-import 'package:gtc_rider/utils/one_signal_config.dart';
 
-class HomePage extends GetView<Locale>  {
+class HomePage extends GetView<Locale> {
```
Comment (fahjan) [inline lib/features/home/presentation/pages/home_page.dart:5]: rename the Locale class to LocaleCotroller

**Review by fahjan — APPROVED** [review]

**Pull request merged**